### PR TITLE
Feat/accessibility filter

### PR DESCRIPTION
**Pull request by TheCraftedGem**
## Summary
What is this branch for what does it do? Adds ability to filter location by accessibilty.

**Zube Card Referenced** 143

**Files Modified**
- `location_search.rb, location_search_spec.rb, location_index.rb`: Added 'English' to the list of languages

### Tests to Run
- `spec/searches/location_search_spec.rb`: Tests the function `hello_world`

### CODE REVIEW

File: lib/tasks/accessibility_categories.rake [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# namespace :accessibilty do
+#   task generate: %i[:accessibilty :deaf_interpreter :disabled_parking :ramp :wheelchair :tape_braille]
```
Comment (justuseapen) [inline lib/tasks/accessibility_categories.rake:2]: remove this if we're not using it

File: data/icarol-csv/categories.csv [hunk 2]
```diff
@@ -516,3 +516,9 @@ bom,id,name,taxonomy_id,parent_id,parent_name,type,filter,filter_parent,filter_p
 ,515,Cell-Phone Allowed,132-08,132,Substance Use Treatment Preferences,situation,TRUE,FALSE,
 ,516,Case Management Services Provided,132-09,132,Substance Use Treatment Preferences,situation,TRUE,FALSE,
 ,517,Support Groups Services Provided,132-10,132,Substance Use Treatment Preferences,situation,TRUE,FALSE,
+,518,Accessibility,99,,,service,TRUE,TRUE,90
```
Comment (justuseapen) [inline data/icarol-csv/categories.csv:519]: remove these

File: lib/tasks/accessibility_categories.rake (hunk 1 above)
Comment (TheCraftedGem) [reply lib/tasks/accessibility_categories.rake:2]: Removed

Comment (TheCraftedGem) [reply lib/tasks/accessibility_categories.rake]: Removed

File: data/icarol-csv/categories.csv (hunk 2 above)
Comment (TheCraftedGem) [reply data/icarol-csv/categories.csv]: Removed

**Review by justuseapen — APPROVED** [review]

**Pull request merged**